conf/master

```diff
@@ -1267,14 +1282,15 @@
 #event_match_type: startswith
 
 # Save runner returns to the job cache
-#runner_returns: True
+#runner_returns: False
 
 # Permanently include any available Python 3rd party modules into thin and minimal Salt
 # when they are generated for Salt-SSH or other purposes.
 # The modules should be named by the names they are actually imported inside the Python.
 # The value of the parameters can be either one module or a comma separated list of them.
 #thin_extra_mods: foo,bar
 #min_extra_mods: foo,bar,baz
+#
```

Comment:
why are we adding this extra comment?

Reply:
Again, because according to the comment at the top of the file, options which are the default ones, should be followed by a blank line - the ones which are not, shouldn't be. Otherwise it is very confusing and the comment at the top doesn't make sense/is redundant.